modules/nodes-nodes-audit-policy-custom.adoc

```diff
@@ -12,7 +12,7 @@ These custom rules take precedence over the top-level profile field. The custom
 
 [IMPORTANT]
 ====
-Custom rules are ignored if the top-level profile field is set to `None`.
+If you set the top-level profile field to `None`, an API server, such as the Kubernetes API server, ignores custom rules and applies its most restrictive policy that disables audit logging.
```

Comment:
I had to re-read this a few times to understand what it meant - just wondering if it's necessary to mention "applies its most restrictive policy". Could it be simplified to just:

```suggestion
If you set the top-level profile field to `None`, an API server, such as the Kubernetes API server, ignores custom rules and disables audit logging.
```